spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns a success response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('This is a list of users')
+    end
```

Comment:
- [ ] [OPTIONAL] Nice work so far, however, as stated above the project requirements states that your Spec file should include tests that ensures correct view template is rendered. I am making this optional because it was missed by the previous reviewer. 

            Example : expect(response).to render_template(:index) 

Kindly update your project accordingly to match this. :pray: